docs/CHANGELOG.asciidoc

```diff
@@ -61,6 +62,37 @@ Review important information about the {kib} 8.x releases.
 * <<release-notes-8.0.0-alpha1>>
 
 --
+
+[[release-notes-8.13.1]]
+== {kib} 8.13.1
+
+The 8.13.1 release includes the following bug fixes.
+
+[float]
+[[enhancement-v8.13.1]]
+=== Enhancements
+Fleet::
+* Feature/default fields ({kibana-pull}178020[#178020]).
```

Comment:
```suggestion
* Remove `index.query.default_field` setting from managed component template settings ({kibana-pull}178020[#178020]).
```
This edit was added to the Fleet Release Notes so I guess we should port it here, too.